pallets/identity/src/lib.rs

```diff
@@ -2298,14 +2297,24 @@ impl<T: Trait> IdentityTrait<T::AccountId> for Module<T> {
 
     /// Adds systematic CDD claims.
     fn unsafe_add_systematic_cdd_claims(targets: &[IdentityId], issuer: SystematicIssuers) {
-        targets.iter().for_each(|new_member| {
-            Self::unsafe_add_claim(
-                *new_member,
-                Claim::CustomerDueDiligence,
-                issuer.as_id(),
-                None,
-            )
-        });
+        // Generate Investor UID for systematic identities.
+        let cdd_ids = targets
+            .iter()
+            .map(|sys_id| CddId::new(sys_id.clone(), InvestorUID::from(&sys_id)))
+            .collect::<Vec<_>>();
+
+        // Add a CDD claim with its systematic UID for each sytematic identity.
+        targets
+            .iter()
+            .zip(cdd_ids.into_iter())
+            .for_each(|(new_member, cdd_id)| {
+                Self::unsafe_add_claim(
```

Comment:
It might be cleaner to combine this into something like
```
targets
    .iter()
    .for_each(|new_member| {
        let cdd_id = CddId::new(new_member.clone(), InvestorUID::from(&new_member))
        Self::unsafe_add_claim(
```
Infact, I'll prefer using `for new_member in targets` instead of `targets.iter().for_each()`